eth/tracers/api.go

```diff
@@ -870,8 +870,8 @@ func (api *API) TraceCall(ctx context.Context, args ethapi.TransactionArgs, bloc
 	}
 	// try to recompute the state
 	reexec := defaultTraceReexec
-	if config != nil && config.Reexec != nil {
```

Comment:
I don't get it. Any difference?

Reply:
To expand more on Gary's comment: `TraceConfig` is embedded in the `TraceCallConfig` struct, so its fields can be accessed directly. See here for some explanation: https://gobyexample.com/struct-embedding.

Please revert this line and then we're good to go!

Reply:
oh thanks I didn't knew about that! 
It's done